books/epub.py

```diff
@@ -149,13 +150,58 @@ def get_info(self):
         return self._info
 
     def get_cover_image_path(self):
```

Comment:
Can you please add some try..catch blocks when returning cover_image (and not `None`), that basically check that the file actually exists `os.path.exists`, and if not, return `None`? This would ward us against epubs that seem to point to a cover, but don't really have the image file, and basically save some trouble on functions that call this one.